### PR TITLE
created news page

### PR DESCRIPTION
**Pull request by Ksenaaa**

### CODE REVIEW

File: src/components/NewsInfo/NewsInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+import styles from './NewsInfo.module.scss'
+
+type Props = {
+  id: string
+}
+
+export const NewsInfo: FC<Props> = ({ id }) => {
+  const { data: news, isLoading, isSuccess, isError } = useGetOneNewsQuery(id)
+  const { data: allnews } = useGetAllNewsQuery()
```
Comment (vell28) [inline src/components/NewsInfo/NewsInfo.tsx]: allNews

File: src/components/NewsInfo/NewsInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+  const indexItem = allnews?.findIndex(item => item.id === id) as number
```
Comment (vell28) [inline src/components/NewsInfo/NewsInfo.tsx]: allnews?.findIndex(item => item.id === id) || 0

File: src/components/NewsInfo/NewsInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+  const previousId = useCallback(() => {
```
Comment (vell28) [inline src/components/NewsInfo/NewsInfo.tsx]: you need rework logic as we discussed

File: src/components/NewsInfo/NewsInfo.tsx (hunk 1 above)
Comment (Ksenaaa) [reply src/components/NewsInfo/NewsInfo.tsx]: fixed

File: src/components/NewsInfo/NewsInfo.tsx (hunk 2 above)
Comment (Ksenaaa) [reply src/components/NewsInfo/NewsInfo.tsx]: fixed

File: src/components/NewsInfo/NewsInfo.tsx (hunk 3 above)
Comment (Ksenaaa) [reply src/components/NewsInfo/NewsInfo.tsx]: fixed

Comment (vell28) [reply src/components/NewsInfo/NewsInfo.tsx]: useMemo

**Review by vell28 — APPROVED** [review]

File: src/components/NewsInfo/NewsInfo.tsx (hunk 3 above)
Comment (Ksenaaa) [reply src/components/NewsInfo/NewsInfo.tsx]: ok

**Pull request merged**